sdk/storage/AzureStorageBlob/Source/Data/Transfer.swift

```diff
@@ -49,7 +49,7 @@ public struct TransferProgress {
 // MARK: Protocols
 
 /// Object that contains information about a transfer operation.
-public protocol Transfer: AnyObject {
+public protocol Transfer: AzureTask, AnyObject {
```

Comment:
I think, per our guidelines, even our non-transfer calls like `listBlobs` need to return an AzureTask so they can be canceled... which, I guess, in this context just means we would ignore any results that would come back if the cancel was issued during the networking transit time.

Reply:
Yes, that's totally correct. However I started working on implementing that and it became a pretty large refactor so I broke it out into issue #325. Perhaps when working on that issue we can find a way to somehow cancel in-flight network requests.